### PR TITLE
[eclipse] initial contribution build fix

### PR DESCRIPTION
**Pull request by marechal-p**
- Fix the build caused by a miss-checked file left in previous commit.
- Fix runtime issues in browsers without wasm support.

Signed-off-by: Paul Maréchal <paul.marechal@ericsson.com>

---

Theia is ready to go now !

### CODE REVIEW

**Comment by elaihau** [comment]
Tested in my local environment. 

`yarn` used to fail with giving the following error, while it runs properly with the changes in this PR. 

screenshot from 2018-06-22 12-16-35

File: packages/core/src/browser/browser.ts [hunk 1]
```diff
@@ -30,7 +30,9 @@ export const isWebKit = (userAgent.indexOf('AppleWebKit') >= 0);
 export const isChrome = (userAgent.indexOf('Chrome') >= 0);
 export const isSafari = (userAgent.indexOf('Chrome') === -1) && (userAgent.indexOf('Safari') >= 0);
 export const isIPad = (userAgent.indexOf('iPad') >= 0);
-export const isNative = typeof (window as any).process !== 'undefined';
+export const isNative = typeof (window as any).process !== 'undefined'; // tslint:disable-line:no-any
```
Comment (marcdumais-work) [inline packages/core/src/browser/browser.ts]: Is this really needed? Does using 'any' without the tslint exception now fails the build>

File: packages/core/src/browser/browser.ts [hunk 2]
```diff
@@ -30,7 +30,9 @@ export const isWebKit = (userAgent.indexOf('AppleWebKit') >= 0);
...
+
+export const isBasicWasmSupported = typeof (window as any).WebAssembly !== 'undefined'; // tslint:disable-line:no-any
```
Comment (marcdumais-work) [inline packages/core/src/browser/browser.ts]: ditto

**Comment by marcdumais-work** [comment]
After discussion, Paul will re-submit a new version of this PR, that addresses only the build issue. Further improvements will be submitted separately, after the first Eclipse submission.

**Comment by marechal-p** [comment]
It was just based on the fact that master was not building, and even after building support for almost anything aside Chrome would throw errors because of wasm non-support. Reverted the runtime fixes so that this PR only holds the build fix.

**Pull request merged**